pycm/pycm_curve.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+"""Curve module."""
+from __future__ import division
+from .pycm_error import pycmCurveError
+from .pycm_util import threshold_func, thresholds_calc, isfloat
+from .pycm_param import *
+from .pycm_obj import ConfusionMatrix
+import numpy
+
+
+class Curve:
+    """
+    Curve class.
+    """
+
+    def __init__(self, actual_vector, prob_vector, classes, thresholds=None, sample_weight=None):
+        """
+        Init method.
+
+        :param actual_vector: actual vector
+        :type actual_vector: python list or numpy array of any stringable objects
```

Comment:
Why not `list` instead of `python list`?

Reply:
To emphasize that it's a Python native list. We used exactly this pattern in the `ConfusionMatrix` and `Compare` classes.